package.json

```diff
@@ -749,17 +749,17 @@
             {
                 "command": "language-julia.clearCurrentInlineResult",
                 "key": "Escape",
-                "when": "editorTextFocus && !editorHasSelection && !findWidgetVisible && !markerNavigationVisible && !parameterHintsVisible && !inSnippetMode && !isInEmbeddedEditor && !suggestWidgetVisible && !onTypeRenameInputVisible && !renameInputVisible && juliaHasInlineResult && (editorLangId == julia || editorLangId == juliamarkdown || editorLangId == markdown)"
```

Comment:
It makes sense that `markdown` can be safely included here because of the julia-specific guard `juliaHasInlineResult`, but I'm not sure it's a good idea to include `markdown` for the other keybindings. (Some included it before, some didn't.)

Best would be to restrict the binding to only when we are inside a proper julia code fence in the case of `markdown`. I recall from messing with syntaxes a few years ago that there is a way to get julia-specific code highlighting in markdown for julia code fences. I wonder if there is an analogous mechanism for the when clause.

Reply:
Unfortunately there's no context key for the injected grammar, so that's not an option.

All other keybindings already included markdown before; the only change here is that certain menu entries will now also appear in markdown files. Imho that's a reasonable behaviour.

Reply:
We could try to detect ourselves whether we are in a Julia code block and then use another custom context for that, I guess? But lets merge this as is for now.